tests/simulation/test_simulator.py

```diff
@@ -132,7 +134,7 @@ def test_new_syncing_peer(self):
         for miner in miners:
             miner.stop()
 
-        self.simulator.run_until_complete(600)
+        self.simulator.run_until_complete(2000)
```

Comment:
Why do we need more time here? It's almost 3.33x more. Is sync-v2 slower to sync in this case?

Reply:
Reverted.